contracts/GatewayVM.sol

```diff
@@ -57,8 +57,8 @@ library GatewayVM {
             ret := 1 // assume zero
             for {
 
-            } lt(p, e) { // while (p < e)
-
+            } lt(p, e) {
+                // while (p < e)
```

Comment:
Why is this comment in the post block?

Reply:
I don't know how prettier and/or vscode works, but whatever it's doing, it formats it that way.  It mangles comment very frequently.  If you can fix the formatting settings, go for it.

In general, I disagree with prettier defaults on almost everything.

Reply:
This is super stupid XD. I've moved the comment above the block.